doc/releases/reef.rst

```diff
@@ -4,6 +4,27 @@ Reef
 
 Reef is the 18th stable release of Ceph. It is named after the reef squid (Sepioteuthis).
 
+v18.2.7 Reef
+============
+This is the seventh backport (hotfix) release in the Reef series. We recommend that all users update to this release.
+
+Notable Changes
+---------------
+
+This rlease has the following OSD issues fixed:
+https://github.com/ceph/ceph/pull/61653
```

Comment:
This release fixes a critical bluestore regression in versions 18.2.5 and 18.2.6 (https://github.com/ceph/ceph/pull/61653). Users running either of those releases are recommended to upgrade at the earliest convenience.
This release also includes several other important BlueStore fixes:
...